server/tidb_serial_test.go

```diff
@@ -210,6 +210,7 @@ func TestTLSBasic(t *testing.T) {
 }
 
 func TestTLSVerify(t *testing.T) {
+	t.Skip("remove skip after upgrading go1.20.1")
```

Comment:
this PR will upgrade to 1.20.1, when should we remove this skip

Reply:
After merging this PR. I will update the CI image. then I will remove it.